Features/final version

Wahaj-Ali opened the pull request:
Here is the link to the video that discusses the description of the project: https://www.loom.com/share/41c4ac7608b04e86acaa282476b965ca

Wahaj-Ali committed:
created navbar and headline section for mobile version

in this commit
- created index.html
- added navbar and headline section for the mobile version.

Wahaj-Ali committed:
created program section for mobile-version

in this commit
- added program section for the mobile version
- added related icons and images to the images folder

Wahaj-Ali committed:
completed home page for mobile version

in this commit
- added speakers section in the home-page of the mobile version.
- added related images and icons

Wahaj-Ali committed:
added a suitable title for index.html

in this commit
- added a title for index.html(home page)

Wahaj-Ali committed:
Create about.html

in this commit
- added about.html to the branch

Wahaj-Ali committed:
updated about.html

in this commit
- set up about page
- added related images and icons
- setup the navbar and some headline section elements

Wahaj-Ali committed:
added headline section for the about page

in this commit
- added headline section for about.html
- added related images
- styled about.html using styles1.css

Wahaj-Ali committed:
completed about page for mobile version

in this commit
- added logo section to the about page
- added events section to the about page
- added partner section to the about page
- added footer at the end.

Wahaj-Ali committed:
set up home page for the desktop version

in this commit
- used media query to set up home page for desktop version

Wahaj-Ali committed:
completed about page for the desktop version

in this commit
- added navbar for the desktop version of about page
- used media query to hide/display items in the about page
- styled about page for the desktop version

Wahaj-Ali committed:
created a final branch with dynamic speaker section

in this commit
- completed the hamburger functionality for the mobile versions for both home and about pages.
- Edited the speaker section and made it dynamically using JavaScript.

Wahaj-Ali committed:
added a descriptive README file

in this commit
- added a descriptive readme file
- made some changes to the code to make it work when deployed

Wahaj-Ali committed:
made changes to the code for deploying the project correctly

Wahaj-Ali committed:
changed path for hamburger icon

Wahaj-Ali committed:
changed directory to the images

Wahaj-Ali committed:
made directory changes

Wahaj-Ali committed:
made changes to the paths of images

Wahaj-Ali committed:
made changes to image paths

Wahaj-Ali committed:
updated readme with acknowledgment to the author.

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
Update linters.yml

in this commit
- updated linters file for scss

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
Update linters.yml

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed further linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
Update styles1.css

removed linter error

Olamarx commented on about.html:
```diff
@@ -0,0 +1,96 @@
+        <img id="circle" src="images/common/circle.png" alt="">
+        <div class="about-intro">
+            <p>35th International Pak Orthocon 2022 is a comprehensive academic activity aimed at a diverse audience consisting of Orthopaedic surgeons, residents and Allied health specialists. Over the years, this Pakistan Orthopaedic Association Annual meeting has become the leading academic activity of the country in the field of Orthopaedics with diverse academic content and renowned National ...
+        </div>
+    </section>
```
- [x] There should be contact details here as it is on the page you followed.
![image](https://user-images.githubusercontent.com/86200356/203583493-4af23a15-8141-4a83-87de-995e6b3095aa.png)

Olamarx commented on about.html:
```diff
@@ -0,0 +1,96 @@
+        <div id="logo-div">
+            <img id="logo" src="images/common/logo.png" alt="">
+        </div>
+    </section>
```
- [x] Kindly follow the pattern used on the project you followed. The divider should come before the text.

### Your design
![image](https://user-images.githubusercontent.com/86200356/203584345-daa0c159-b0b7-4946-ad46-9673e82354f5.png)

### Behance
![image](https://user-images.githubusercontent.com/86200356/203584585-3bdf0450-b396-401c-9841-229c4b35f57a.png)

Olamarx commented on about.html:
```diff
@@ -0,0 +1,96 @@
+    <!-- event time section -->
+    <section class="event">
+        <h2>Pak Orthocon 2022 Icon</h2>
+        <p>Following is the icon for this years International Orthocon</p>
```
- [x] Kindly center this tag appropriately
![image](https://user-images.githubusercontent.com/86200356/203585250-3b5e5fd5-183e-42f9-9356-645072f394b1.png)

Olamarx commented on index.html:
```diff
@@ -0,0 +1,148 @@
+
+    <!-- Main program -->
+    <section class="main-program">
+        <h1>Scientific Program</h1>
```
- [x] Kindly use another hx as h1 is professionally and conventionally advised to be used once. This is because h1 carries a message of title.

Olamarx commented on index.html:
```diff
@@ -0,0 +1,148 @@
+         <h2>Featured Speakers</h2>
+        <hr>
+        <div class="speaker-s">
+        <!-- <div class="speaker">
```
[OPTIONAL] I would advise you to remove the commented code so that your file would be easy to read.

Olamarx commented on about.html:
```diff
@@ -0,0 +1,96 @@
+        <div class="ham" class="cross"></div>
+        <!-- <a id="hamburger" href=""><img  src="images/mobile/hamburger-icon.svg" alt=""></a> -->
+
+        <a id="nav-logo" href="index.html"><img src="images/common/logo.png" alt=""></a>
```
- [x] Kindly add alt and not leave it empty so that other devises can access the content.

Olamarx commented on index.js:
```diff
@@ -0,0 +1,76 @@
+},
+];
+
+function displaySpeakers() {
```
[OPTIONAL] Consider using ES6 arrow function as it would help you in React module and even make an up-to-date dev.

Olamarx reviewed (changes requested):
Hi @Wahaj-Ali, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media4.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- All linters passed ✔️ 
- Beautiful UI 👍🏼 
- Loom explanation 👍🏼 

## Required Changes ♻️
_Check the comments under the review._

- [ ] I think you forgot to add this top aspect which has login and co. of the site as it is in Behance

![image](https://user-images.githubusercontent.com/86200356/203580293-337f8ac8-a3ab-4c1e-8cef-491830c4e571.png)

- [ ] Kindly use the latest readme file as it has been updated. You can find it [here](https://github.com/microverseinc/readme-template)

- [ ] The width of the desktop version is going overboard. You will notice it by the right hand side. I think you need to give it a max-width. 
![image](https://user-images.githubusercontent.com/86200356/203594774-a64b4948-ed5b-48e4-92b3-e18c077a252e.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Wahaj-Ali committed:
made required changes

in this commit
- added navbar with Facebook and twitter logos
- edited alt tags
- made changes to the about page to match the template.

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
removed linter errors

Wahaj-Ali committed:
Update styles.css

danmainah commented on README.md:
```diff
@@ -1 +1,41 @@
-# Capstone-1
\ No newline at end of file
+
```
- [x] @Wahaj-Ali as the previous reviewer indicated kindly change your readme file to the latest one, here is the link to [the template](https://github.com/microverseinc/readme-template)

danmainah reviewed (changes requested):
Hi @Wahaj-Ali , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Wahaj-Ali committed:
Update README.md

Wahaj-Ali committed:
Update README.md

Wahaj-Ali committed:
Create LICENSE.md

Wahaj-Ali committed:
updated README

Wahaj-Ali committed:
Merge branch 'features/final-version' of https://github.com/Wahaj-Ali/Capstone-1 into features/final-version

Wahaj-Ali committed:
Update README.md

Qoosim reviewed (approved):
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Wahaj-Ali, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 

## Top Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 

![](https://user-images.githubusercontent.com/34779928/191333120-c0c2ce83-28e1-4dbc-aac1-09e467f7a8bd.gif)


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
